docs/proposals/formatted-output.md

```diff
@@ -92,13 +92,14 @@ oras manifest fetch $REGISTRY/$REPO:$TAG --format json
 ```
 
 > [!NOTE]
-> The formatted output of `oras manifest fetch` only supports OCI/Docker Image Manifest in this release. [OCI image index](https://github.com/opencontainers/image-spec/blob/v1.1.0-rc5/image-index.md) and other media types of artifact will be supported in future releases.
+> - The formatted output of `oras manifest fetch` only supports OCI/Docker Image Manifest in this release. [OCI image index](https://github.com/opencontainers/image-spec/blob/v1.1.0-rc5/image-index.md) and other media types of artifact will be supported in future releases.
```

Comment:
@qweeah Technically, it is trivial to support any JSON-based manifest after we supporting json output of image manifests, isn't it?

Reply:
It is trivial to support known JSON-based manifest types. I am not sure what do you mean by any. E.g. if we are going to support ORAS artifacts(application/vnd.cncf.oras.artifact.manifest.v1+json), then [code of the manifest](https://github.com/oras-project/artifacts-spec/blob/main/specs-go/v1/manifest.go) must be added to ORAS CLI.